documentation/source/users/rmg/running.rst

```diff
@@ -4,33 +4,107 @@
 
     python rmg.py input.py -p
 
 Run with multiprocessing for reaction generation and QMTP::
 
     python rmg.py -n <Max number of processes allowed> input.py 
 
+Run with setting a limit on the maximum execution time::
+
+	python rmg.py input.py -t <DD:HH:MM:SS>
+
+Run with setting a limit on the maximum number of iterations::
+
+	python rmg.py input.py -i <Max number of desired iterations>
+
 We recommend you make a job-specific directory for each RMG simulation. Some jobs can take quite a while to complete, so we also recommend using a job scheduler if working in a linux environment. 
 
 
 Details on the multiprocessing implementation
-----------------------------------------------
+---------------------------------------------
 
 Currently, multiprocessing is implemented for reaction generation and the generation of QMfiles when using the QMTP option to compute thermodynamic properties of species. The processes are spawned and closed within each function. The number of processes is determined based on the ratio of currently available RAM and currently used RAM. The user can input the maximum number of allowed processes from the command line. For each reaction generation or QMTP call the number of processes will be the minimum value of either the number of allowed processes due to user input or the value obtained by the RAM ratio. The RAM limitation is employed, because multiprocessing is forking the base process and the memory limit (SWAP + RAM) might be exceeded when using too many processors for a base process large in memory.
 
 In python 3.4 new forking contexts 'spawn' and 'forkserver' are available. These methods will create new processes which share nothing or limited state with the parent and all memory passing is explicit. Once RMG is transferred to python 3 it is recommended to use the spawn or forkserver forking context to potentially allow for an increased number of processes.
 
+
+Details on profiling RMG jobs
+-----------------------------
+
+Here, we explain how to profile an RMG job. For starters, use the ``saveSeedModulus`` option in the input file, as described in the Section :ref:`Miscellaneous Options <miscellaneousoptions>`, to save the seed mechanism at regular intervals, perhaps every 50 or 100 iterations depending on the size of the mechanism. This flag is particularly important for saving intermediate steps when working with large mechanisms; it may be prudent to save and examine how the chemistry changes over mechanism development rather than just obtaining the final seed mechanism.
+
+
+Additionally, these seeds can then be restarted with use of the ``-i`` flag, as described in the Section :ref:`Input Flags <inputflags>` above. Restarting these seeds allows examination of how computational effort, time spent in each module, individual processor memory consumption if using the the ``-n`` flag, and overall memory consumption change over the course of mechanism development. To time profile, one could use::
```

Comment:
Talking about restarting seeds, I thought you would refer to the `-r` option. But then the example is about something else. Do you mean to discuss seeds being restarted, or are you talking about something else? Sorry if I seem obtuse, I'm just a bit confused.

Reply:
Thanks for the review! I see that I could’ve explained that better. The full syntax that I should’ve included for these two examples are:

`rmg.py restart_from_seed.py -r <path_to_seed>/seed -p -i 15`

and 

`mprof run --multiprocess rmg.py -n 3 restart_from_seed.py -r <path_to_seed>/seed -p -i 15`

My previous use of `input.py` was incorrect since this paragraph was discussing how to pair the profiling tools with a restart job. I additionally wasn’t clear regarding my omission of the `-r` flag. Since the default argument for `-r` is an empty string, by default RMG looks for the required `seed` directory for a restart job in the same directory as the user is running `restart_from_seed.py`. To help me be organized, I often place the `seed` directory in the same directory as `restart_from_seed.py`, which allows me to omit the `-r` flag by taking advantage of its default argument. I’ve updated this PR with the proper syntax. 


Please let me know if I can update this further to be more clear. We’d love to have people understand the restart feature and new profiling tools 🙂